SideBar

Polosmakrr opened the pull request:
Базова розмітка та стилі для SideBar

Polosmakrr committed:
SideBar

Базова розмітка та стилі

AegisVP commented on src/components/SideBar/SideBar.js:
```diff
@@ -0,0 +1,35 @@
+import { Section, Container, List, Title, Item, Text } from './SideBarComponents';
+
+export const SideBar = () => {
```
Додай одразу пропси щоб вираховувати показники

Polosmakrr committed:
Update logic SideBar

Добавив базову логіку, вона не точна після того як буде точна структура даних то я перероблю

AegisVP committed:
change images location
